release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -506,6 +506,16 @@ For more information, see xref:../scalability_and_performance/cnf-numa-aware-sch
 
 For more information about the feature, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-using-pgt-to-configure-power-saving-states_ztp-advanced-policy-config[Configuring power states using PolicyGenTemplates CRs].
 
+[id="ocp-4-13-scalability-and-performance-talm-updates"]
+==== Pre-caching container images for updates
+
+For container image pre-caching, {cgu-operator-first} compares the available disk space with the estimated {product-title} image size to ensure that there is enough space. 
```

Comment:
Suggested rewrite which clarifies that there are two new features in TALM.

```suggestion
This release adds two new {cgu-operator-first} features for use with GitOps ZTP:
 
* A new check ensures that there is sufficient available disk space on the managed cluster host before cluster updates.
Now, during container image pre-caching, {cgu-operator} compares the available host disk space with the estimated {product-title} image size to ensure that there is enough disk space on the host. 

* A new `excludePrecachePatterns` field in the `ConfigMap` CR is available that controls which pre-cache images {cgu-operator} downloads to the cluster host before an update. 
```

Reply:
agreed